presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -128,41 +135,29 @@ public void testPublic()
     @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
     public void testTableOwnerPrivileges()
     {
-        String tableName = "alice_table";
-        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
-
-        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
-
         onHive().executeQuery("set role admin;");
-        assertThat(onHive().executeQuery(format("SHOW GRANT USER alice ON TABLE %s", tableName)).
-                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Select only the relevant columns of SHOW GRANT
+        assertThat(onHive().executeQuery(format("SHOW GRANT USER %s ON TABLE %s", PRINCIPAL_NAME, tableName)).
+                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Skip unknown columns such as "grant_time" of SHOW GRANT
                 ImmutableList.of(
-                        row("default", tableName, "alice", "USER", "SELECT", Boolean.TRUE, "alice"),
-                        row("default", tableName, "alice", "USER", "INSERT", Boolean.TRUE, "alice"),
-                        row("default", tableName, "alice", "USER", "UPDATE", Boolean.TRUE, "alice"),
-                        row("default", tableName, "alice", "USER", "DELETE", Boolean.TRUE, "alice")
+                        row(DATABASE, tableName, PRINCIPAL_NAME, PRINCIPAL_TYPE, "SELECT", GRANT_OPTION, GRANTOR),
```

Comment:
I disagree. What if you change the tablename or principal name in one of the tests but not the other? They aren't really static constants, they're expected results that happen to be the same, and it's ok to have some duplication.